packages/manager/modules/manager-pci-common/src/components/logs/LogHowTo.component.tsx

```diff
@@ -69,64 +63,47 @@ export function LogHowTo({ onGotoStreams }: Readonly<LogHowToProps>) {
     }
   };
 
+  const getLogButtonLabel = () => {
+    if (!hasAccount) {
+      return t('log_button_create_account');
+    }
+    if (hasAccount && !hasStream) {
+      return t('log_button_create_stream');
+    }
+    return t('log_button_transfer_stream');
+  };
+
   return (
     <>
-      <OsdsText
-        color={ODS_THEME_COLOR_INTENT.primary}
-        level={ODS_THEME_TYPOGRAPHY_LEVEL.heading}
-        size={ODS_TEXT_SIZE._300}
-      >
-        {t('log_tile_section_title')}
-      </OsdsText>
-      <OsdsTile className="mt-4" color={ODS_THEME_COLOR_INTENT.primary}>
+      <OdsText preset="heading-1">{t('log_tile_section_title')}</OdsText>
+      <OdsCard className="mt-4" color={ODS_THEME_COLOR_INTENT.primary}>
```

Comment:
`colore` property can be removed, that's the default value for new `Card` component: 
https://ovh.github.io/design-system/latest/?path=/docs/ods-components-card--technical-information